agent/engine/docker_container_engine.go

```diff
@@ -317,6 +317,10 @@ func (dg *DockerGoClient) createContainer(ctx context.Context, config *docker.Co
 		return DockerContainerMetadata{Error: CannotGetDockerClientError{version: dg.version, err: err}}
 	}
 
+	// Hardcode the network we are going to use (xplenty/overlay)
+	// For this agent to work we must create this network (once) first.
+	hostConfig.NetworkMode = "xplenty"
```

Comment:
@leonidlm - Just wondering - couldn't we pass this to `docker create` call as part of it's host configuration file?